blog/content/post/2023-09-12-simplifying-firmware-update.md

```diff
@@ -103,9 +124,14 @@ on-board peripherals.
 
 Capsule Updates are also supported under Linux by the fwupd / LVFS project, as
 well as various BSD derivatives, support for which we helped develop in the
-past few years. TODO link to relevant blogposts
+past few years:
+[fwupd for FreeBSD status update](https://blog.3mdeb.com/2021/2021-06-14-fwupd-freebsd-status/)
```

Comment:
Missing period at the end of the sentence but overall this blog post is well-written and informative! Good job @mkopec.

I did have some corrections to the abstract and title. Sorry, I didn't look into the abstract previously: https://github.com/3mdeb/news-and-ideas/pull/538#discussion_r1338643023, and https://github.com/3mdeb/news-and-ideas/pull/538#discussion_r1338642946.

Reply:
Fixed in d1426a8ca9f2b738428b08d4aef5d07590b382dc